src/pages/CreateClass/index.js

```diff
@@ -1,10 +1,37 @@
 import React from "react";
-
 import CreateClassComponent from '../../components/Class'
+import ClassesList from '../../components/Class/ClassesList'
+import Modal from '../../components/common/Modal'
+import "../../components/Class/ClassesList/styles.scss"
+
 
 function CreateClass() {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
```

Comment:
In my view the better name could be `toggleClassFormModal`.  what do you think?

Reply:
Done